modules/minimint-wallet/src/db.rs

```diff
@@ -73,7 +74,22 @@ pub struct UnsignedTransactionKey(pub Txid);
 impl DatabaseKeyPrefixConst for UnsignedTransactionKey {
     const DB_PREFIX: u8 = DB_PREFIX_UNSIGNED_TRANSACTION;
     type Key = Self;
-    type Value = PartiallySignedTransaction;
+    type Value = UnsignedTransaction;
+}
+
+#[derive(Clone, Debug, Encodable, Decodable)]
+pub struct UnsignedTransaction {
+    pub psbt: PartiallySignedTransaction,
+    pub signatures: Vec<(PeerId, PegOutSignatureItem)>,
+}
+
```

Comment:
It's a bit unusual to also have the value struct in `mod db`. So far there are mostly keys in there (that are really only needed for the DB and nothing else)

Reply:
Gotcha, I saw the `PendingTransaction` struct there as well so I thought that might be the pattern but I could move both to `lib`.